usr/sbin/pkcscca/pkcscca.c

```diff
@@ -648,16 +655,21 @@ CK_FUNCTION_LIST *p11_init(void)
 {
     CK_RV rv;
     CK_RV (*getfunclist)(CK_FUNCTION_LIST_PTR_PTR ppFunctionList);
-    char *loc1_lib = "/usr/lib/pkcs11/PKCS11_API.so64";
-    char *loc2_lib = "libopencryptoki.so";
+
+#if defined(_AIX)
+    char *loc1_lib = LIB_PATH "/pkcs11/PKCS11_API.so";
+#else
+    char *loc1_lib = LIB_PATH "/pkcs11/PKCS11_API.so64";
```

Comment:
wow, I guess the .so64 is crap (and has been crap before already).
I guess we can change this to just use OCK_API_LIBNAME, no need for trying loc1 first.